crates/nu-protocol/src/pipeline_data.rs

```diff
@@ -169,7 +179,11 @@ impl PipelineData {
                         }
                     }
                     Value::String {
-                        val: output,
+                        val: if trim_end_newline {
+                            output.trim_end_matches(LINE_ENDING).to_string()
```

Comment:
I don't fully like the use of `str::trim_end_matches(&'a self, P) -> SomeIterator<'a>`.

- Is it agreed upon behavior to strip all consecutive trailing newlines or just the last?
- This unnecessarily clones most of the string. We could instead only remove the end without reallocating. (Factor out into function or extension trait)

Reply:
> Is it agreed upon behavior to strip all consecutive trailing newlines or just the last?

I've tried fish shell behavior, and it trip all consecutive all trailing newlines.

> This unnecessarily clones most of the string. We could instead only remove the end without reallocating. (Factor out into function or extension trait)

Thanks!  That's a good point

Reply:
Thanks for the quick fix!

> I've tried fish shell behavior, and it trip all consecutive all trailing newlines.

OK